src/test/testing-library/SwaggerComponents.test.tsx

```
@@ -0,0 +1,97 @@
+import Navigation from '../../features/swagger/components/Navigation/Navigation';
+import useSwagger from '../../features/swagger/hooks/useSwagger';
+
+const backToTheHome: string = t('navigation.navigate_to_home_page');
+const arrowAlt: string = t('navigation.back_arrow_description');
+
+describe('Swagger Navigation', () => {
+  const originalLocation: Location = window.location;
+  const mockAssign: jest.Mock = jest.fn();
+
+  beforeAll(() => {
+    delete (window as { location?: Location }).location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)